tests/testthat/test-target_market_share.R

```diff
@@ -468,3 +468,28 @@ test_that("w/ some region missing some scenario outputs expected `production`
   expect_equal(target_sds$global$production, 0.5)
   expect_equal(target_sds$`non opec`$production, 0.5)
 })
+
+test_that("w/ no matching regions, outputs empty named tibble", {
+
```

Comment:
Nit: This spaces will be gone once we pass the styler. But it's nice to not have them in the first place.